external/rxbit/Tools/carthage-copy-and-verify

```diff
@@ -1,19 +0,0 @@
-#!/bin/sh
```

Comment:
Does this mean we'll no longer support developer mode (aka building dependencies from sources)?

Reply:
I will ask more questions about it in direct chat, thanks